Automattic/wp-calypso
# VideoPress Onboarding: Add 'vendor' prop for better domain search results.

#### Proposed Changes

* Passed the `videopress` vendor string to the `RegisterDomainStep` component to get search results matching video-related domains better.

Old results:
<img width="866" alt="Screen Shot 2022-10-07 at 1 12 19 PM" src="https://user-images.githubusercontent.com/789137/194637540-986423f5-4140-4030-85cd-cea80c8d094d.png">

New results:
<img width="849" alt="Screen Shot 2022-10-07 at 1 11 09 PM" src="https://user-images.githubusercontent.com/789137/194637596-03369161-b603-4aba-9337-93ef103d22b4.png">

#### Testing Instructions



* Start at http://calypso.localhost:3000/setup/intro?flow=videopress
* Complete the flow until you get to the domain search step.
* Search for a domain, and it should offer you video-related domains like .movie, .video, etc.

#### Pre-merge Checklist
